configure.ac

```diff
@@ -1457,32 +1457,32 @@ AS_CASE([$host],
   [[powerpc64*-*-linux-musl*]],
     [has_native_backend=yes; arch=power; model=ppc64; system=linux],
   [[s390x*-*-linux*]],
-    [has_native_backend=yes; arch=s390x; model=z10; system=linux],
+    [has_native_backend=yes; arch=s390x; system=linux],
   # expected to match "gnueabihf" as well as "musleabihf"
   [armv6*-*-linux-*eabihf],
-    [arch=arm; model=armv6; system=linux],
+    [arch=arm; system=linux],
   [armv7*-*-linux-*eabihf],
-    [arch=arm; model=armv7; system=linux],
+    [arch=arm; system=linux],
   [armv8*-*-linux-*eabihf],
-    [arch=arm; model=armv8; system=linux],
+    [arch=arm; system=linux],
   [armv8*-*-linux-*eabi],
-    [arch=arm; model=armv8; system=linux],
+    [arch=arm; system=linux],
   [armv7*-*-linux-*eabi],
-    [arch=arm; model=armv7; system=linux],
+    [arch=arm; system=linux],
   [armv6t2*-*-linux-*eabi],
-    [arch=arm; model=armv6t2; system=linux],
+    [arch=arm; system=linux],
   [armv6*-*-linux-*eabi],
-    [arch=arm; model=armv6; system=linux],
+    [arch=arm; system=linux],
   [armv6*-*-freebsd*],
-    [arch=arm; model=armv6; system=freebsd],
+    [arch=arm; system=freebsd],
   [earmv6*-*-netbsd*],
-    [arch=arm; model=armv6; system=netbsd],
+    [arch=arm; system=netbsd],
   [earmv7*-*-netbsd*],
-    [arch=arm; model=armv7; system=netbsd],
+    [arch=arm; system=netbsd],
   [armv5te*-*-linux-*eabi],
-    [arch=arm; model=armv5te; system=linux],
+    [arch=arm; system=linux],
   [armv5*-*-linux-*eabi],
-    [arch=arm; model=armv5; system=linux],
+    [arch=arm; system=linux],
   [arm*-*-linux-*eabihf],
     [arch=arm; system=linux],
   [arm*-*-linux-*eabi],
```

Comment:
If the goal is to simplify the configure script, you could probably collapse all these ARM cases into three (linux / freebsd / netbsd).

Reply:
I had considered collapsing them, but the existing style in that chunk is to never collapse, see e.g. `arm64-*-darwin*` and `aarch64-*-darwin*` being separate `case` entries with the same settings.